File: 9. JS Front-End/Conditional Statements/PasswordGuess.js
function passwordGuess(input) {

    let secretPass = "s3cr3t!P@ssw0rd";

    let password = input[0];

    if (password == secretPass) {
        console.log("Welcome")
    } else {
        console.log("Wrong password!")
    }
}

passwordGuess(["s3cr3t!P@ssw0rd"])

File: Programming basics with Java/Problem conditions/PasswordGuess.java
import java.util.Scanner;

public class PasswordGuess {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        String inputPass = scanner.nextLine();

        String secretPass = "s3cr3t!P@ssw0rd";

        if (inputPass.equals(secretPass)) {
            System.out.println("Welcome");
        } else {
            System.out.println("Wrong password!");
        }
    }
}
